executor/scripts/build_local.py

```
@@ -198,28 +210,33 @@ def _download_windows_claude_binary() -> tuple[str, str] | None:
+        f"Looking for {target_platform} Claude CLI binary (sdk version: {sdk_version or 'latest'})..."
     )
 
+    platform_tag = platform_tags.get(target_platform)
+    if not platform_tag:
+        print(f"Error: Unknown target platform: {target_platform}")
+        return None
+
     try:
         # Query PyPI API for package info
         pypi_url = "https://pypi.org/pypi/claude-agent-sdk/json"
         with urllib.request.urlopen(pypi_url, timeout=30) as response:
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)